### PR TITLE
Deleted trash in main

### PR DESCRIPTION
**Pull request by tagirshin**
With love

### CODE REVIEW

**Comment by tagirshin** [comment]
Прифффки)

**Comment by tagirshin** [comment]
Added new method

File: main.py [hunk 1]
```diff
@@ -1,11 +1,2 @@
 def hello(name):
     return 'hello, hello' + name
```
Comment (tansunasyrova) [inline main.py]: I want docstrings

File: second.py [hunk 2]
```diff
@@ -1,4 +1,5 @@
-
+def minus(a, b):
```
Comment (tansunasyrova) [inline second.py:3]: change your code, please

**Review by tansunasyrova — CHANGES_REQUESTED** [review]

**Pull request merged**